features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Endpoints/GeneralConfiguration/Certificates.jsx

```diff
@@ -59,7 +59,7 @@ const useStyles = makeStyles(theme => ({
     },
     certificatesHeader: {
         fontWeight: 600,
-        marginTop: 20,
+        marginTop: '10px',
```

Comment:
Shall we take these values from theme object?

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/7535/commits/419a0f5140ea1caa34abf2805f4f28937054cfd5